src/plugins/intel_cpu/src/ngraph_transformations/convert_logsoftmax.cpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "convert_logsoftmax.hpp"
+#include <openvino/opsets/opset1.hpp>
+#include <openvino/opsets/opset8.hpp>
+#include <openvino/core/rt_info.hpp>
+#include <openvino/pass/pattern/op/wrap_type.hpp>
+#include <openvino/pass/pattern/op/or.hpp>
+#include <transformations/utils/utils.hpp>
+
+#include "itt.hpp"
+
+ov::intel_cpu::ConvertLogSoftmax::ConvertLogSoftmax() {
+    MATCHER_SCOPE(ConvertLogSoftmax);
+    using namespace ov::pass::pattern;
+    auto logSoftmax = wrap_type<ov::opset8::LogSoftmax>({any_input()});
+    matcher_pass_callback callback = [=](Matcher& m) {
+        const auto& pattern_map = m.get_pattern_value_map();
+        auto logSoftmaxNode = ov::as_type_ptr<ov::opset8::LogSoftmax>(pattern_map.at(logSoftmax).get_node_shared_ptr());
+        auto axis = std::make_shared<opset8::Constant>(ov::element::i64, ov::Shape{}, logSoftmaxNode->get_axis());
+        auto input = logSoftmaxNode->get_input_node_ptr(0);
+        auto xMax = std::make_shared<opset8::ReduceMax>(input->get_default_output(), axis->get_default_output(), true);
```

Comment:
In case if `input` is an operation with several outputs we get error "Default output not supported" here. So I suggest to add input to the matcher and use matched output here.
Minor: also, I suppose we don't need to call get_default_output() method for the operations that exactly have only one input (this is true for all operations created in this transformation). So I suggest to make the code cleaner
```suggestion
        auto input = pattern_map.at(input_m);
        auto xMax = std::make_shared<opset8::ReduceMax>(input, axis, true);
```

Reply:
fix in 621fcd0a2171c863eb60b44608cea93871506381